docs/sources/getting-started/glossary.md

```diff
@@ -40,12 +40,30 @@ This topic lists words and abbreviations that are commonly used in the Grafana d
       Explore allows a user to focus on a building a query. User can refine the query to return the expected metrics before building a dashboard.
     </td>
   </tr>
+  <tr>
+    <td style="vertical-align: top">Exporter</td>
+    <td>
+      An exporter implements the translation of data that comes out of a data source into a format that Prometheus can digest.
+    </td>
+  </tr>
+  <tr>
+    <td style="vertical-align: top">Integration (Grafana Cloud)</td>
+    <td>
+      Each Integration in Grafana Cloud uses the cloud agent to connect your data source to Grafana Cloud for visualizing. Note: Prometheus uses the word “integrations” to refer to software that exposes Prometheus metrics without needing an exporter, which is a different use of the same word we use here.
```

Comment:
Should Cloud Agent be capitalized?

Reply:
No. I'm trying to avoid the unnecessary proliferation of proper nouns when descriptive ones serve the purpose. I may lose that battle, but haven't today.